components/core/dock.tsx

```diff
@@ -18,6 +18,7 @@ import {
   useMemo,
   useRef,
   useState,
+  React
```

Comment:
`isValidElement` here, and update `React.isValidElement` to `isValidElement` below